cmd/utils/flags.go

```diff
@@ -212,10 +212,10 @@ var (
 		Name:  "snapshot",
 		Usage: `Enables snapshot-database mode (default = enable)`,
 	}
-	TxLookupLimitFlag = cli.Int64Flag{
+	TxLookupLimitFlag = cli.Uint64Flag{
 		Name:  "txlookuplimit",
-		Usage: "Number of recent blocks to maintain transactions index by-hash for (default = index all blocks)",
-		Value: 0,
+		Usage: "Number of recent blocks to maintain transactions index by-hash for (default = index about one year)",
```

Comment:
Does `0` mean `infinity` ? Might need a clarification in the Usage

Reply:
Yeah, there's no way to say infinity in Go, so 0 is kind of "disable"

Reply:
Right, but `0`, for a uint-flag, might to the user mean "Don't index anything". And if the user sets `0` with that notion, it's bad UX

Reply:
Alternatives:
- keep it as a `int` flag, and make `-1` mean "everything"
- Add to teh Usage string: `(default = index about one year, 0 = index everything)`

Reply:
Went with the 2nd suggestion